<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rigPublishWin</class>
 <widget class="QMainWindow" name="rigPublishWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PT rig publish</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Current Scene : </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="scene_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Asset Name : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="asset_label">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0,0,1,0,0,0,0,0">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="6,1,6">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Command list</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QListWidget" name="cmd_listWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QListWidget" name="exe_listWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Excecute List</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>cmd path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cmd_lineEdit"/>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Preset : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="preset_comboBox"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="refresh_pushButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="clear_pushButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="add_pushButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_pushButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Name : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="preset_lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="savePreset_pushButton">
            <property name="text">
             <string>Save Preset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Logs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="log_textEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Publish Task</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="task_comboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="save_checkBox">
          <property name="text">
           <string>Save as</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="output_lineEdit">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ref_radioButton">
          <property name="text">
           <string>Ref</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dev_radioButton">
          <property name="text">
           <string>Dev</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="batch_checkBox">
        <property name="text">
         <string>Batch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="publish_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Run Command</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="workFile_pushButton">
        <property name="text">
         <string>Open work file</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="outputFile_pushButton">
        <property name="text">
         <string>Open output file</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
